### ISSUE TITLE
Skins and Launcher not Loading

### ISSUE THREAD

**Issue by deadbushmc** [ChristopherHX/mcpelauncher-manifest#310]
So I installed the launcher in my Ubuntu 20.04
but I click on the launcher and it dosent wanna open.
I used appimage and made opened with terminal but there i got an issue that custom skin wont work
please fix it or tell me a way to fix the launcher :>

**Comment by ChristopherHX** [comment]
> tell me a way to fix the launcher

Yes please tell me, if it would be easy then it would work already.

To get marketplace, skin loading, crafting, textures working
- edit profile
- click the version
- select Minecraft 1.16.201 or older

**Comment by deadbushmc** [comment]
> > tell me a way to fix the launcher
> 
> Yes please tell me, if it would be easy then it would work already.
> 
> To get marketplace, skin loading, crafting, textures working
> 
>     * edit profile
> 
>     * click the version
> 
>     * select Minecraft 1.16.201 or older

The launcher itself isnt launching

**Comment by deadbushmc** [comment]
and textures are broken too
![image](https://user-images.githubusercontent.com/66917109/134770999-96965a69-2334-435b-ae73-3a093194afd8.png)

**Comment by deadbushmc** [comment]
@ChristopherHX umm what
![image](https://user-images.githubusercontent.com/66917109/134771132-860a78c8-3b4a-44ac-8d83-dbb408bbebd0.png)
![image](https://user-images.githubusercontent.com/66917109/134771160-2866537a-3d12-403e-8152-5b5a44a778fa.png)
cant craft what 
![image](https://user-images.githubusercontent.com/66917109/134771172-129f6a5f-3982-4037-adc4-1af68b54f9f4.png)

**Comment by ChristopherHX** [comment]
I thought you told me the launcher doesn't open at all!

> and textures are broken too

> cant craft what

Same for all android/x86(_64) devices, while smartphones doesn't have this bug.

**Marketplace / Skins loading is a won't fix from my side**, I don't need that.

**Comment by deadbushmc** [comment]
> I thought you told me the launcher doesn't open at all!
> 
> > and textures are broken too
> 
> > cant craft what
> 
> Same for all android/x86(_64) devices, while smartphones doesn't have this bug.
> 
> **Marketplace / Skins loading is a won't fix from my side**, I don't need that.

See The launcher won't open the one I installed with flatpak and Idek how to uninstall it ot only launched once and skin was loading. But I got appimage and made it executable in that I cannot add skin and the crafting is bugged.

**Comment by technicaldiff** [comment]
There is a workaround to the Crafting issue which affect intel x86 devices
Workaround is to switch UI profile to classic then right click in recipe book instead of left clicking
Reference: https://bugs.mojang.com/browse/MCPE-117105

**Comment by technicaldiff** [comment]
> So I installed the launcher in my Ubuntu 20.04 but I click on the launcher and it dosent wanna open. I used appimage and made opened with terminal but there i got an issue that custom skin wont work please fix it or tell me a way to fix the launcher :>

for the skin open a singleplayer world apply your skin and the leave it will work.You will have to reapply every time you relaunch
